File: Classes/CoreData/Objects/CDContent.m
//
//  CDContent.m
//  Dokie
//
//  Created by Ashamaz Shidov on 18/03/2019.
//  Copyright © 2019 Ashamaz Shidov. All rights reserved.
//

#import "CDContent.h"

@implementation CDContent

@dynamic data;
@dynamic document;

#pragma mark -
static NSString *_kData = @"data";
+ (NSString *)kData { return _kData; }

static NSString *_kDocument = @"document";
+ (NSString *)kDocument { return _kDocument; }

@end

File: Classes/CoreData/Objects/CDContent.swift
//
//  CDContent.swift
//  Dokie
//
//  Created by Ashamaz Shidov on 16.05.2020.
//  Copyright © 2020 Ashamaz Shidov. All rights reserved.
//

import CoreData

class CDContent : NSManagedObject {
    
    @NSManaged var data: Data
    @NSManaged var document: CDDocument?

    static let kData = "data"
    static let kDocument = "document"

}
